llvm/include/llvm/CodeGen/StackProtector.h

```diff
@@ -57,26 +109,14 @@ class StackProtector : public FunctionPass {
 
   /// The minimum size of buffers that will receive stack smashing
   /// protection when -fstack-protection is used.
-  unsigned SSPBufferSize = DefaultSSPBufferSize;
+  unsigned SSPBufferSize = SSPLayoutInfo::DefaultSSPBufferSize;
 
   // A prologue is generated.
   bool HasPrologue = false;
 
   // IR checking code is generated.
   bool HasIRCheck = false;
```

Comment:
These fields look like they were moved to a new location, but not removed from the old one?

Reply:
I don't follow this. If you moved them the old references should be deleted? Changing less code isn't preferable to leaving dead code